portfolio/src/main/webapp/index.html

```diff
@@ -9,13 +9,10 @@
   <header>
     <div id ="header">
         <h1 style="text-align:center"> Sean Tisdale Portfolio </h1>
```

Comment:
move the h1 style to the stylesheet too.

you could probably just do:

h1 {
 ... styles here
} if you aren't using h1 elsewhere

Reply:
solved